app/components/nav-bar.js

```diff
@@ -8,5 +8,21 @@ export default Component.extend({
       this.get('authManager').logout();
       this.get('routing').transitionTo('index');
     }
+  },
+
+  didInsertElement() {
+    this._super.call(this);
+    this.$('.notification.item').popup({
+      popup : '.popup',
+      on    : 'click'
+    });
+  },
+
+  willDestroyElement() {
+    this._super.call(this);
+    this.$('.notification.item').popup({
```

Comment:
@CosmicCoder96 why do you even need a separate css for color ?

This does all the required things. And no additional css added.

```html
<div class="item">
    <div class="header">
        New Session Proposal for event1 by  user1
    </div>
    <div class="content weight-600">
        Title of proposal
    </div>
    <div class="left floated content">
        just now
    </div>
</div>
```